### DISCUSSION TITLE
Proxmox update includes GPU Drivers, will it impact Frigate LXC?

### DISCUSSION THREAD

**Discussion by marleyjaffe** [community-scripts/ProxmoxVE#835] [Q&A]
I have frigate LXC installed and working with GPU passthrough. I want to update my proxmox environment but am concerned since the upgrade seems to also update the nvidia GPU drivers. Is this a valid concern? What can I do proactively to ensure everything will keep working or will be easily fixed/reverted? It took me tooo long to get frigate working and want as minimal pain as possible. Thanks for the insights!

**Comment by michelroegl-brunner** [comment]
Best to ask at the Firgate Github to check with the maintainers if a GPU Driver update can cause issues.

https://github.com/blakeblackshear/frigate

**Comment by remz1337** [comment]
Hey, on my end, I have been updating my Nvidia drivers across LXCs (including Frigate) for some time without any issue. The important thing is to have the exact same version between the host and the LXC. First update the host, then the LXCs, the reboot the server.

BUT, better safe than sorry, so please backup your stuff :)
